packages/doenetml/src/Viewer/renderers/graph.jsx

```diff
@@ -116,6 +116,24 @@ export default React.memo(function Graph(props) {
                 }
             }
         });
+
+        // If an object is below the navigation bar and it is clicked, both
+        // the navigation button and the object drag/action are triggered.
+        // We hack our way around this behaviour by preventDefault-ing on all the button events
+        // after the nav bar is created.
+        const preventDefault = (e) => {
+            e.preventDefault();
+        };
+        newBoard.containerObj
+            .querySelectorAll(".JXG_navigation_button")
+            .forEach((button) => {
+                button.addEventListener("click", preventDefault);
+                button.addEventListener("touchstart", preventDefault);
+                button.addEventListener("mousedown", preventDefault);
+                button.addEventListener("mouseup", preventDefault);
+            });
+
+        window.BOARD = newBoard;
```

Comment:
It doesn't appear that setting `window.BOARD` is doing anything. Plus, it would get overwritten if there were more than one graph in a document.

Reply:
Woops, this is not supposed to be there.